ci/jobs/README.md

```diff
@@ -75,3 +75,65 @@ Once you have the UUID of interest, add it into your job template. Each job temp
 or more private credentials to be installed in the `ssh-agent` using the `ssh-agent-credentials`
 directive. See the [`ssh-agent-credentials docs`](http://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.ssh-agent-credentials)
 for examples.
+
+Example Config
+--------------
+
+Create the config file.
+
+```commandline
+mkdir ~/.config/jenkins_jobs/
+vi ~/.config/jenkins_jobs/jenkins_jobs.ini
+```
+
+Here is an example config:
+```ini
+[jenkins]
+user=bbouters
+password=5cxr6230dptzp6l7oqga03chupnqa8f5
+url=https://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com
+query_plugins_info=False
+
+[job_builder]
+include_path=ci/jobs/scripts
+```
+
+Example Usage
+-------------
+
+Test the potential job changes. Ensure sure you are at the top of the repo.
+
+```commandline
```

Comment:
Minor point: `commandline` isn't recognized as a valid language by github or the `markdown-calibre` parsers, nor by any other parser I know of. I think you mean `sh`.